src/serac/infrastructure/about.cpp

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/infrastructure/about.hpp"
+
+#include "serac/serac_config.hpp"
+
+#include "serac/infrastructure/git_sha.hpp"
+
+#include "axom/config.hpp"
+#include "axom/core.hpp"
+#include "axom/fmt.hpp"
+
+#ifdef SERAC_USE_CALIPER
+#include "caliper/caliper-config.h"
+#endif
+
+#ifdef SERAC_USE_CONDUIT
+#include "conduit_config.h"
+#endif
+
+#ifdef SERAC_USE_HDF5
+#include "hdf5.h"
+#endif
+
+#ifdef SERAC_USE_LUA
+#include "lua.h"
+#endif
+
+#include "mfem.hpp"
+
+#ifdef SERAC_USE_RAJA
+#include "RAJA/config.hpp"
+#endif
+
+#ifdef SERAC_USE_UMPIRE
+#include "umpire/Umpire.hpp"
+#endif
```

Comment:
were these manually alphabetized, or was that done by `clang-format`?

Reply:
I did it

Reply:
Though I should put our headers under theirs.